modules/twinkleblock.js

```diff
@@ -1381,7 +1388,8 @@ Twinkle.block.blockGroups = [
 		list: [
 			{ label: 'Advertising', value: 'uw-adblock' },
 			{ label: 'Arbitration enforcement', value: 'uw-aeblock' },
-			{ label: 'Block evasion', value: 'uw-ipevadeblock' },
+			{ label: 'Block evasion - IP', value: 'uw-ipevadeblock' },
+			{ label: 'Block evasion - TA', value: 'uw-tempevadeblock' },
```

Comment:
Can we change TA to Temporary Account? Less jargony. I think folks know the acronym IP, but are less likely to know TA.